packages/utils/src/format.ts

```diff
@@ -1,4 +1,5 @@
-import {toHexString} from "./bytes.js";
+import {keccak256} from "ethereum-cryptography/keccak.js";
+import {fromHex, toHexString} from "./bytes.js";
```

Comment:
careful with these, fromHex does not support browser environments, best to just use the to/fromHex of ssz library

Reply:
Hmm right, dangerous indeed. Maybe we would want to rationalize those low level primitives somehow?

Reply:
or we do something like I did in base64 utils
https://github.com/ChainSafe/lodestar/blob/b5344bfe59cf8b0b7d13ae4722fed2f463506b7b/packages/utils/src/base64.ts#L1

Reply:
This last suggestion sounds like the way to go. As much as possible we should wrap `nodejs` specifics into platform neutral utilities.
This is an issue related to that: https://github.com/ChainSafe/lodestar/issues/6371 (only mentions `process`, but there are issues with `Buffer` too)

Reply:
Can address that in a different PR, for now I'd suggest to use ssz utils